lib/xboxrt/Makefile

```diff
@@ -0,0 +1,17 @@
+SRCS += \
+	$(NXDK_DIR)/lib/xboxrt/libc_extensions/debug.c \
```

Comment:
This doesn't fit here. It's not a typical libc function, and this is one of the APIs I'd like to deprecate and remove asap. I don't see how this is "runtime" code either.

Reply:
It's moved to `/lib/hal` now. I'm aware it's not an ideal place, but it's supposed to go away eventually anyway, just like a lot of the old code in `/lib/hal`.